platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/IconFactory.java

```diff
@@ -24,11 +24,14 @@
 
 /**
  * Factory for creating {@link Icon} objects.
+ *
  * @see Icon
  */
 public final class IconFactory {
 
     private static final String ICON_ID_PREFIX = "com.mapbox.icons.icon_";
+    public static final Bitmap ICON_MARKERVIEW_BITMAP = Bitmap.createBitmap(1, 1, Bitmap.Config.ARGB_8888);
```

Comment:
in 69f30a I opted to use alpha_8 instead, this is smaller and only represents alpha channel, no color information is associated. It results in a size of one byte with value 0, this indicates the bitmap is transparent